client/src/modules/user/model/actions.js

```diff
@@ -149,3 +149,26 @@ export const getAccountDetails = token =>
 
 export const deleteAccount = (email, password) =>
   deleteData('/auth', { email, password });
+
+export const saveEmailNotificationSettings = emailNotificationsFrequency => async dispatch => {
```

Comment:
this is comment only - `reportFrequency` is more adequate, sound shorter and better